hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/CreateHoodieTableCommand.scala

```diff
@@ -120,10 +119,8 @@ object CreateHoodieTableCommand {
 
     val tableType = tableConfig.getTableType.name()
     val inputFormat = tableType match {
-      case DataSourceWriteOptions.COW_TABLE_TYPE_OPT_VAL =>
+      case DataSourceWriteOptions.COW_TABLE_TYPE_OPT_VAL | DataSourceWriteOptions.MOR_TABLE_TYPE_OPT_VAL =>
```

Comment:
@linfey90 i don't think this change makes sense to me. 

Can you please elaborate what you're trying to achieve here?

Reply:
In hive queries, the original table name is used instead of the suffix _rt _ro table name. at this point we will choose to skip the _ro table. I also think hive offline tasks should use optimization table, so its inputFormat default value should be HoodieParquetInputFormat.Also If this default value there are other considerations,I will compare and modify when use hive sync metadata.

Reply:
Surely, it might make sense in your use-case but it doesn't make sense in general use-case. Why would we want to treat MOR tables as COW by default in general?

Reply:
year,I began to think that Hive offline tasks are not sensitive to time and data timeliness. It is better to use read optimized tables and keep the COW InputFormat as the default value. But according to the latest data, reading snapshot data is also good.

Reply:
@alexeykudinkin @linfey90 folks what is the conclusion here? should we close this?